bookmarks.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="styles/index.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Amatic+SC:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body class="app-grid" background="./images/bg.jpeg">
+    <header>
+      <h1>Quizzzle</h1>
+    </header>
+
+    <main class="app-grid__main">
+      <h2 class="h2lines">
+        Your bookmarks
+        <img
+          src="./images/bookmarks.svg"
+          alt="bookmark"
+          width="30px"
+          class="h2lines"
+        />
+      </h2>
+
+      <details class="qbox">
```

Comment:
Not a good classname. Please try to avoid abbreviations. Better would be `quiz-card` or similar.